ext/standard/tests/bug73124.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Bug #73124 (php_ini_scanned_files relied on PHP_CONFIG_FILE_SCAN_DIR)
+--SKIPIF--
+<?php
+if (!empty(PHP_CONFIG_FILE_SCAN_DIR)) die("Skip: PHP_CONFIG_FILE_SCAN_DIR must not be available");
```

Comment:
I'm not sure I understand why this skip exists ?

Reply:
Simply because the bug is caused by PHP_CONFIG_FILE_SCAN_DIR having to have a value (rather than relying on the existence of `php_ini_scanned_files`). So there didn't seem much point in running the test if that value already exists.

When this is fixed, I intend to submit an actual test for this function (there isn't one) and also for `php_ini_loaded_file` (there is an existing one but it is not very specific).

Reply:
But now the code doesn't care about that constant, why should the test ?

Reply:
True, but it was intended to be a regression/bug test rather than a general one. I can remove the skip if you want, but then how does your test reporting stuff know that `bug73124.phpt` if actually a test for `php_ini_scanned_files`? Advice please.